packages/custom-templated-path-webpack-plugin/test/index.js

```diff
@@ -32,7 +32,9 @@ describe( 'CustomTemplatedPathPlugin', () => {
 	} );
 
 	it( 'should resolve with basename output', async () => {
-		await webpackAsync( config );
-		await accessAsync( outputFile );
+		expect( async () => {
```

Comment:
This actually meant to verify that it's not throwing in the meantime, so we're adding that assertion to make it a full-featured test.